i18n/ru/docusaurus-plugin-content-docs/current/reference/isolation/coupling-cohesion.md

```diff
@@ -16,7 +16,7 @@ sidebar_position: 1
 
 ## Композиция компонентов (UI level) {#components-composition-ui-level}
 
-Абсолютное большинство современных UI-фреймоворков и библиотек предоставляют компонентную модель, в которой каждый компонент может иметь собственные свойства, собственное состояние и дочерние компоненты, а также, зачастую, слоты.
+Абсолютное большинство современных UI-Фреймворков и библиотек предоставляют компонентную модель, в которой каждый компонент может иметь собственные свойства, собственное состояние и дочерние компоненты, а также, зачастую, слоты.
```

Comment:
question (non-blocking): why the capitalization?

Reply:
Oops. Seems like it should be brought back :)